src/button/buttonview.js

```diff
@@ -39,19 +40,19 @@ export default class ButtonView extends View {
 			],
 
 			on: {
-				mousedown: ( evt ) => {
+				mousedown: bind.to( evt => {
 					evt.preventDefault();
-				},
+				} ),
 
-				click: () => {
+				click: bind.to( () => {
```

Comment:
Would be awesome to use as `bind.if( 'isEnabled', callback )`. If that's a big thing, please report it as a feature request in ckeditor5-ui.